roles/nginxplus/templates/default_nginx.conf.j2

```diff
@@ -0,0 +1,69 @@
+{{ ansible_managed | comment }}
+
+{% if nginx_modules.njs %}
+{% if nginx_http_template_enable %}
+load_module modules/ngx_http_js_module.so;
+{% endif %}
+{% if nginx_stream_template_enable %}
+load_module modules/ngx_stream_js_module.so;
+{% endif %}
+{% endif %}
+{% if nginx_modules.perl %}
+load_module modules/ngx_http_perl_module.so;
+{% endif %}
+{% if nginx_modules.geoip %}
+{% if nginx_http_template_enable %}
+load_module modules/ngx_http_geoip_module.so;
+{% endif %}
+{% if nginx_stream_template_enable %}
+load_module modules/ngx_stream_geoip_module.so;
+{% endif %}
+{% endif %}
+{% if nginx_modules.image_filter %}
+load_module modules/ngx_http_image_filter_module.so;
+{% endif %}
+{% if nginx_modules.rtmp and nginx_type == "plus" %}
+load_module modules/ngx_rtmp_module.so;
+{% endif %}
+{% if nginx_modules.xslt %}
+load_module modules/ngx_http_xslt_filter_module.so;
+{% endif %}
```

Comment:
I would remove the `if/endif` sections for modules we do not use and do not plan to use, just to clean up the template and make it easier to read (and also to reduce the number of variables we need).